nginx.conf

```diff
@@ -27,6 +27,27 @@ http {
     listen <%= ENV["PORT"] %>;
     server_name localhost;
 
+    # Note: .+ instead of .*
+    # this is to allow /data/ to pass through untouched
+
+    location ~* ^/data/(.+) {
```

Comment:
This nginx proxy rule takes anything that begins with `/data/`, and has **at least one more subsequent character** after that, and proxies it to the S3 bucket we set up through a Cloud Foundry service associated with the production app.

The `/data/` page will continue to render the `/data/index.html` file generated by Jekyll, while anything inside that will proxy to the bucket.